json4s/src/test/scala/org/http4s/json4s/Json4sSpec.scala

```diff
@@ -1,14 +1,14 @@
 package org.http4s
 package json4s
 
+import cats.syntax.option._
```

Comment:
I have an object, `org.http4s.batteries._`, which imports all cats syntax and instances.  I'm going to be maintaining the scalaz-stream port for a while for work.  By using that batteries trait, I can isolate many of the differences in there and keep the two branches from diverging so much.

But at this point, I'm happy with whatever makes more things compile and work. :)